code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -101,4 +101,119 @@
 		attack_verb = list("hit", "poked")
 
 	playsound(src.loc, 'sound/weapons/batonextend.ogg', 50, 1)
-	add_fingerprint(user)
\ No newline at end of file
+	add_fingerprint(user)
+
+
+//Hotline Miami-inspired weapons
+/obj/item/weapon/psycho

```

Comment:
remove this comment
why do these weapons need a separate subtype? add them under weapon/melee